### ISSUE TITLE
Number of points detected per kymograph varies

### ISSUE THREAD

**Issue by clairelbromley** [dougkelly88/ClaireDataAnalysis#77]
Just wondering whether this was expected?

For this edge detection, only 18 points are on the graph. Then the corresponding edge that comes up on the viewer is two pixels out (i.e. two pixels away from the position where the scattered light ends)
![240815 embryo 11 cut 4 kymograph index along cut 6 - quantitative kymograph](https://cloud.githubusercontent.com/assets/13903697/13294876/f82b19ba-db1d-11e5-8464-c608e453279a.png)
![screen shot 2016-02-24 at 17 43 12](https://cloud.githubusercontent.com/assets/13903697/13294907/1bd215b2-db1e-11e5-85a0-50e9b5fcd8e5.png)

Then, for this edge detection in the same embryo, 19 points are detected. This results in the edge detection being "off" by one pixel
![240815 embryo 11 cut 4 kymograph index along cut 7 - quantitative kymograph](https://cloud.githubusercontent.com/assets/13903697/13294980/6cc3268c-db1e-11e5-93ed-659e3f053f6f.png)
![screen shot 2016-02-24 at 17 47 16](https://cloud.githubusercontent.com/assets/13903697/13295029/b402f4b4-db1e-11e5-93af-b8922d552b8b.png)

**Comment by dougkelly88** [comment]
Yes - determined by the condition that the Canny edge-detected region that is identified as the membrane front is the highest region (i.e. closest to the cut) that extends more than 80% across the width of the 4 second quantitative kymograph region

**Comment by dougkelly88** [comment]
Consistently missing pixel at the start of the identified front is odd - perhaps an artefact of the canny edge detection (smoothing?). Won't make a difference under assumption of linear response but will lead to an underestimation of speed if response is non-linear (i.e. steeper initially after the cut).
